images/miq-app/Dockerfile

```diff
@@ -100,7 +100,6 @@ RUN ${APPLIANCE_ROOT}/setup && \
     mv /etc/httpd/conf.d/ssl.conf{,.orig} && \
     echo "# This file intentionally left blank. ManageIQ maintains its own SSL configuration" > /etc/httpd/conf.d/ssl.conf && \
     cp ${APP_ROOT}/config/cable.yml.sample ${APP_ROOT}/config/cable.yml && \
-    echo "export APP_ROOT=${APP_ROOT}" >> /etc/default/evm && \
     echo "export CONTAINER=true" >> /etc/default/evm
```

Comment:
I don't think we can do that. systemd resets the environment when it starts a unit so I'm afraid moving this would prevent the application from seeing it.

I didn't test specifically with this variable, but when I moved the ruby PATH change to just be in the ruby Dockerfile the evmserverd.service failed to start because it couldn't find `rake`.

I don't think it's worth coding around that if we are trying to get rid of systemd anyway.